### PR TITLE
Fix wrong index access method in PostgreSQL \d \di slash commands

### PR DESCRIPTION
**Pull request by kozikowskik**
Changes related to Issue #235. These changes fix the wrong index type returned by PostgreSQL slash commands.

### CODE REVIEW

File: drivers/metadata/metadata.go [hunk 1]
```diff
@@ -466,14 +467,15 @@ func (i IndexSet) Get() *Index {
...
-	Columns   string
+	Catalog      string
+	Schema       string
+	Table        string
+	Name         string
+	IsPrimary    Bool
+	IsUnique     Bool
+	Type         string
+	Columns      string
+	AccessMethod string
```
Comment (nineinchnick) [inline drivers/metadata/metadata.go:478]: Is there a case when both index type and access method are used at the same time? If not, maybe we shouldn't add a new field here and only return access method as the type in the postgresql driver? The query that fetched this should use COALESCE

Comment (kozikowskik) [reply drivers/metadata/metadata.go]: I added suggested changes.

File: drivers/metadata/postgres/metadata.go [hunk 2]
```diff
@@ -277,11 +277,24 @@ SELECT
   c.relname as "Name",
   CASE i.indisprimary WHEN TRUE THEN 'YES' ELSE 'NO' END,
   CASE i.indisunique WHEN TRUE THEN 'YES' ELSE 'NO' END,
-  CASE c.relkind WHEN 'r' THEN 'table' WHEN 'v' THEN 'view' WHEN 'm' THEN 'materialized view' WHEN 'i' THEN 'index' WHEN 'S' THEN 'sequence' WHEN 's' THEN 'special' WHEN 'f' THEN 'foreign table' WHEN 'p' THEN 'partitioned table' WHEN 'I' THEN 'partitioned index' END as "Type"
+  COALESCE(am.amname, 
+  	CASE c.relkind 
+		WHEN 'r' THEN 'table'
```
Comment (nineinchnick) [inline drivers/metadata/postgres/metadata.go:282]: Nit: there's a condition below to only include `relkind` of `i`, `I` and empty. Can you remove other cases here?

**Review by nineinchnick — APPROVED** [review]

File: drivers/metadata/metadata.go [hunk 3]
```diff
@@ -466,14 +466,14 @@ func (i IndexSet) Get() *Index {
...
-	Type      string
-	Columns   string
+	Catalog      string
+	Schema       string
+	Table        string
+	Name         string
+	IsPrimary    Bool
+	IsUnique     Bool
+	Type         string
+	Columns      string
```
Comment (nineinchnick) [inline drivers/metadata/metadata.go:476]: Revert this change
